src/main/scala/mesosphere/marathon/state/Group.scala

```diff
@@ -232,10 +241,18 @@ object Group extends StrictLogging {
   def emptyUpdate(id: PathId): raml.GroupUpdate = raml.GroupUpdate(Some(id.toString))
 
   /** requires that apps are in canonical form */
-  def validNestedGroupUpdateWithBase(base: PathId): Validator[raml.GroupUpdate] =
+  def validNestedGroupUpdateWithBase(base: PathId, originalRootGroup: RootGroup): Validator[raml.GroupUpdate] =
     validator[raml.GroupUpdate] { group =>
       group is notNull
 
+      // Only top-level groups are allowed to set the enforce role parameter.
+      if (group.enforceRole.contains(true)) {
```

Comment:
Why do we only check for true and not isDefined? Wouldn't that allow one to set the enforceValue to "false" on all groups?

Reply:
Yes, but `false` is the default.